ccd-definition/CaseEventToFields/CareSupervision/manageOrders/6_reviewSection.json

```diff
@@ -23,5 +23,31 @@
     "PageDisplayOrder": 7,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersCloseCase",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": "review",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "orderTempQuestions.closeCase=\"YES\"",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersCloseCaseInformation",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "READONLY",
+    "PageID": "review",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "orderTempQuestions.closeCase=\"YES\" AND manageOrdersCloseCase=\"Yes\"",
```

Comment:
Could we standardise the case in these? i.e. either upper-case or camel-case?

Reply:
I agree this is not a very nice line to look at with a "YES" and a "Yes". The Yes and No option UI returns a "Yes" so I think it would probably best if we changed orderTempQuestions to use "Yes" for consistency.

I think this would require changes in a lot of places which might result in a lot of merge conflicts so it might best to make that change when we clean up the manage orders event at some point, I think that might also be better for testing